test/SlowTests/Server/Documents/ETL/Raven/BasicRavenEtlTests.cs

```diff
@@ -69,6 +72,73 @@ public void Simple_script()
             }
         }
         
+        [Fact]
+        public async Task EtlCanSendDocWithTimeSeriesAndAttachment()
```

Comment:
We need a similar test for counters + attachments and counters + attachments + time series

Reply:
Please put it in a dedicated file `RavenDB_19696.cs`

Reply:
done